RevenueCat/Scripts/EntitlementInfo.cs

```diff
@@ -26,15 +28,15 @@ public EntitlementInfo(JSONNode response)
             IsActive = response["isActive"].AsBool;
             WillRenew = response["willRenew"].AsBool;
             PeriodType = response["periodType"];
-            LatestPurchaseDate = FromUnixTime(response["latestPurchaseDateMillis"].AsLong);
-            OriginalPurchaseDate = FromUnixTime(response["originalPurchaseDateMillis"].AsLong);
+            LatestPurchaseDate = FromUnixTimeInMilliseconds(response["latestPurchaseDateMillis"].AsLong);
```

Comment:
the old code just happened to be working because we were sending seconds in these `millis` fields, so fixing the `millis` fields ironically broke this